docs/configuration/arguments.md

```diff
@@ -343,15 +343,16 @@ It also talks to a KVStore and has it's own copies of the same flags used by the
 
    Setting this to `true` enables writing to WAL during ingestion.
 
-- `-ingester.checkpoint-enabled`
-   Set this to `true` to enable checkpointing of in-memory chunks to disk. This is optional which helps in speeding up the replay process.
-
 - `-ingester.checkpoint-duration` 
    This is the interval at which checkpoints should be created.
 
 - `-ingester.recover-from-wal`
    Set this to `true` to recover data from an existing WAL. The data is recovered even if WAL is disabled and this is set to `true`. The WAL dir needs to be set for this.
 
+- `-ingester.checkpoint-enabled`
+   Set this to `true` to enable checkpointing of in-memory chunks to disk. This is optional which helps in speeding up the replay process.
```

Comment:
Now that `true` is the default, reading the doc looks a bit weird. Maybe we can rephrase it explaining when should be set to false? If there's no good reason to set it to false, why don't we remove the `-ingester.checkpoint-enabled` from this page at all?

Reply:
I don't mind removing it from the docs. I will update the flag description to communicate when to set it to false.